core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3802,8 +3802,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String MASTER_JOURNAL_TEMPORARY_FILE_GC_THRESHOLD_MS =
         "alluxio.master.journal.temporary.file.gc.threshold";
 
-    public static final String MASTER_RPC_FORKJOIN_POOL_PARALLELISM =
-        "alluxio.master.rpc.fork.parallelism";
+    public static final String MASTER_RPC_THREAD_POOL_PARALLELISM =
+        "alluxio.master.rpc.threadpool.parallelism";
```

Comment:
nit: why  `alluxio.master.rpc.threadpool.parallelism` not `alluxio.master.rpc.threadpool.size` which seems easier to remember and type?